doc/nrf/config_and_build/sysbuild_migration.rst

```diff
@@ -0,0 +1,579 @@
+
+Bluetooth fast pair
+###################
+
+Bluetooth fast pair hex generation is now handled by sysbuild, the following Kconfig options exist:
+
++------------------------------------------+-------------------------------------------+
+| Kconfig option                           | Description                               |
++==========================================+===========================================+
+| :kconfig:option:`SB_CONFIG_BT_FAST_PAIR` | Enables Bluetooth fast pair functionality |
++------------------------------------------+-------------------------------------------+
+
+Applications must set this at a sysbuild level if they want the Bluetooth fast pair data generated, the method of supply the key data via command line remains unchanged as it worked in previous |NCS| versions.
+
+.. _child_parent_to_sysbuild_migration_matter:
+
+Matter (Connected home over IP)
+###############################
+
+Matter configuration for generating factory data and over-the-air firmware update images is now controlled directly by sysbuild, the following Kconfigs exist:
+
++---------------------------------------------------------------------+---------------------------------------------------+
+| Kconfig option                                                      | Description                                       |
++=====================================================================+===================================================+
+| :kconfig:option:`SB_CONFIG_MATTER`                                  | Enable matter support                             |
++---------------------------------------------------------------------+---------------------------------------------------+
+| :kconfig:option:`SB_CONFIG_MATTER_FACTORY_DATA`                     | Generate factory data                             |
++---------------------------------------------------------------------+---------------------------------------------------+
+| :kconfig:option:`SB_CONFIG_MATTER_FACTORY_DATA_MERGE_WITH_FIRMWARE` | Merge factory data with main application firmware |
++---------------------------------------------------------------------+---------------------------------------------------+
+| :kconfig:option:`SB_CONFIG_MATTER_OTA`                              | Generate over-the-air firmware update image       |
++---------------------------------------------------------------------+---------------------------------------------------+
+| :kconfig:option:`SB_CONFIG_MATTER_OTA_IMAGE_FILE_NAME`              | Filename for over-the-air firmware update image   |
++---------------------------------------------------------------------+---------------------------------------------------+
+
+Applications must enable these options if they generate factory data or need an over-the-air firmware update, note that the configuration data for the factory data file is still configured from the main application.
+
+.. _child_parent_to_sysbuild_migration_nrf700x:
+
+nRF700x
```

Comment:
Is there also some nRF54H20-specific info that should be added? @nordicjm

Reply:
No clue, nrf54h20 is out of my scope